dev/ci/scripts/utils/ci_utils.sh

```diff
@@ -2,8 +2,7 @@
 
 # Determine HOMEgfs_ and source machine detection early
 if [[ -z "${HOMEgfs_}" ]]; then
-    SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
-    HOMEgfs_="$("${SCRIPT_DIR}/find_homegfs.py")"
+  HOMEgfs_="$(cd "$(dirname "${BASH_SOURCE[0]}")/../../../.." && pwd)"
```

Comment:
My friend has a suggestion on how to handle this more gracefully:

```
HOMEgfs_="$(git rev-parse --show-toplevel)"
```

I'm OK with moving forward with this as-is.  I'll open another issue to tackle this.